evalml/automl/automl_search.py

```diff
@@ -1145,7 +1145,11 @@ def _post_evaluation_callback(self, pipeline, evaluation_results, job_log):
         cv_data = evaluation_results["cv_data"]
         cv_scores = evaluation_results["cv_scores"]
         is_baseline = pipeline.model_family == ModelFamily.BASELINE
-        cv_score = cv_scores.mean()
+        if len(cv_scores) == 1:
+            validation_score = cv_scores[0]
+            mean_cv_score = np.nan
```

Comment:
If there is only 1 score, it was a TV split. We then set the `mean_cv_score~ to NaN